packages/protocol/engine/notarization/manager.go

```diff
@@ -124,8 +124,17 @@ func (m *Manager) createCommitment(index epoch.Index) (success bool) {
 
 	m.pendingConflictsCounters.Delete(index)
 
-	stateRoot, manaRoot := m.storage.ApplyStateDiff(index, m.storage.LedgerStateDiffs.StateDiff(index))
+	blocksCount := m.EpochMutations.TotalAcceptedBlocks(index)
+	txsCount := m.EpochMutations.TotalAcceptedTransactions(index)
+	validatorsCount := m.EpochMutations.TotalActiveValidators(index)
+	epochDetails := &EvictedEpochDetails{
```

Comment:
I was doing that in the first place, and the details were already forgotten, or at least it seemed so. That's I added this event. Hmm I'll try again, maybe something changed, or I did something incorrectly the first time

Reply:
nah, it will not work in the current state of `commit` function. Maybe the eviction of all the data structures is delayed from the `epochCommited` event but the eviction of the epoch is now triggered right in `packages/protocol/engine/notarization/epochmutations.go:186`. Not sure why this is set this way now. 
I can remove this func from there and trigger eviction of notarization mgr epoch data on `EpochEvicted` event if there is no real purpose behind the current state